tests/test-run-calculation.php

```diff
@@ -0,0 +1,273 @@
+                                        'operator' => '+',
+                                        'field' => 'fld_7465068',
+                                    ),
+                                ),
+                            ),
+                        ),
+                    ),
+                    'manual_formula' => '(%field1%+%field2%)/%field3%',
+                    'email_identifier' => 0,
+                    'personally_identifying' => 0,
+                ),
+            ),
+          'fld_3665187' => 
+          array(
+            'ID' => 'fld_3665187',
+            'type' => 'button',
+            'label' => 'Save',
+            'slug' => 'save',
+            'conditions' => 
+            array(
+              'type' => '',
+            ),
+            'caption' => '',
+            'config' => 
+            array(
+              'custom_class' => '',
+              'type' => 'submit',
+              'class' => 'btn btn-default',
+              'target' => '',
+            ),
+          ),
+        ),
+        'page_names' => 
+        array(
+          0 => 'Page 1',
+        ),
+        'mailer' => 
+        array(
+          'on_insert' => 1,
+          'sender_name' => 'Caldera Forms Notification',
+          'sender_email' => 'nico@calderawp.com',
+          'reply_to' => '',
+          'email_type' => 'html',
+          'recipients' => '',
+          'bcc_to' => '',
+          'email_subject' => 'Calculations',
+          'email_message' => '{summary}
+      <div style="display: none;"></div>
+      <div style="display: none;"></div>
+      <div style="display: none;"></div>',
+        ),
+        'check_honey' => 1,
+        'antispam' => 
+        array(
+          'sender_name' => '',
+          'sender_email' => '',
+        ),
+        'conditional_groups' => 
+        array(
+          '_open_condition' => '',
+        ),
+        'settings' => 
+        array(
+          'responsive' => 
+          array(
+            'break_point' => 'sm',
+          ),
+        ),
+        'privacy_exporter_enabled' => false,
+        'version' => '1.8.4',
+        'db_id' => '45',
+        'type' => 'primary',
+        '_external_form' => 1,
+    );
+
+    /**
+     * @since 1.8.5
+     *
+     * @covers \calderawp\calderaforms::run_calculation()
+     */
+    public function testRunCalculation()
+    {
+        $fieldOneValue = 6;
+        $fieldTwoValue = 4;
+        $fieldThreeValue = 2;
+        $form = $this->form;
+        $field = $form['fields']['fld_8568604'];
+
+        Caldera_Forms::set_field_data('fld_1892141', $fieldOneValue, $form );
+        Caldera_Forms::set_field_data('fld_93143', $fieldTwoValue, $form );
+        Caldera_Forms::set_field_data('fld_7465068', $fieldTwoValue, $form );
+        
+        $result = ($fieldOneValue + $fieldTwoValue) / $fieldThreeValue;
+
+        //Test Calculation by fields
+        $cf_calc = Caldera_Forms::run_calculation( null, $field, $form ); 
```

Comment:
@New0 Small point, but changed `$cf_calc` to `$expectedResult` that would be more obvious when reviewing the test later.